packages/devextreme/js/localization/messages/ar.json

```
@@ -311,6 +311,7 @@
         "dxCalendar-captionCenturyLabel": "Century selection",
         "dxCalendar-selectedDate": "The selected date is {0}",
         "dxCalendar-selectedDateRange": "The selected date range is from {0} to {1}",
+        "dxCalendar-readOnly": "Read-only calendar",
```

Usually we use a noun at the end of a variable name
```suggestion
        "dxCalendar-readOnlyLabel": "Read-only calendar",
```

---

[6e4a276](https://github.com/DevExpress/DevExtreme/pull/27898/commits/6e4a276381dbf4ab16f3ad59c181c9a9d0f85b73)